### DISCUSSION TITLE
Please add Fake TCP support to v2

### DISCUSSION THREAD

**Discussion by mrgspeed** [apernet/hysteria#735] [Ideas]
fake tcp works better and usually has better ping in my country (Iran)

sometimes our firewall fully blocks entire UDP so fake tcp will be super handy

thank you for this amazing tool <3

**Comment by 6p5ra** [comment]
+1

**Comment by su-kaka** [comment]
+1

**Comment by zytakeshi** [comment]
+1 It's also the case for mainland China now.
